src/css/legal.scss

```diff
@@ -1,129 +1,125 @@
 .legal-page {
-    margin-top: 100px;
-
-    a:not(.btn) {
-        color: $secondary-color;
-        text-decoration: none;
+  margin-top: 100px;
+
+  a:not(.btn) {
+    color: $secondary-color;
+    text-decoration: none;
+  }
+
+  h1 {
+    font-size: 48px;
+    line-height: 52px;
+    letter-spacing: -2px;
+    font-weight: 800;
+  }
+
+  h2 {
+    font-family: "Work Sans", sans-serif;
+    font-size: 32px;
+    line-height: 40px;
+    font-weight: 700;
+    letter-spacing: -1px;
+    margin: 3rem 0 1rem;
+    &:first-child {
+      margin: 0 0 2rem;
     }
-
-    h1 {
-        font-size: 48px;
-        line-height: 52px;
-        letter-spacing: -2px;
-        font-weight: 800;
+  }
```

Comment:
Extra closing curly brace. I think this should be removed as it's a part of the removed h1 a few lines above.

Reply:
Thanks for your review, there is nothing changes in there except replace tab size 4 to 2. :)